blocks/rich-text/index.js

```diff
@@ -726,14 +711,15 @@ export class RichText extends Component {
 	}
 
 	updateContent() {
-		const bookmark = this.editor.selection.getBookmark( 2, true );
-		this.savedContent = this.props.value;
-		this.setContent( this.savedContent );
-		this.editor.selection.moveToBookmark( bookmark );
-
-		// Saving the editor on updates avoid unecessary onChanges calls
-		// These calls can make the focus jump
-		this.editor.save();
+		// Do not trigger a change event coming from the TinyMCE undo manager.
+		// Our global state is already up-to-date.
+		this.editor.undoManager.ignore( () => {
```

Comment:
What would be causing an undo level to be added? I added a breakpoint to TinyMCE's internal `UndoManager#add` and it was never triggered as a result of anything which occurs in the callback of this function.

Reply:
When content is set, but not by MCE?

Reply:
But specifically the functions called within this block do not appear to add any undo levels from what I have been able to tell. See also #7620. Could we demonstrate by a failing end-to-end test what we need to be accounting for by its presence if it were to be removed?

Reply:
Worst case there's also a `no_events` argument which can be passed with `setContent` that may be an alternative to what we have here.